app/models/signin.py

```diff
@@ -0,0 +1,58 @@
+import re
+import datetime
+from ..db.db import *
+
+
+class CheckUserData:
+    """
+    class contains methods for data validation
+    """
+    def __init__(self):
+        self.dataError = None
+
+    def checkWhiteSpace(self, items):
+        items = list(items.values())
+        for item in items:
+            if item.isspace():
+                self.dataError = {'result': 'Invalid input(Only whitespace)'}, 405
+                return(self.dataError)
+        return(False)
+
+    def checkEmptyData(self, items):
+        items = list(items.values())
+        for item in items:
+            if not item:
+                self.dataError = {'result': 'Invalid input (Empty data)'}, 405
+                return(self.dataError)
+        return False
+
+    def checkEmail(self, items):
+        items = items
+        match = re.match('^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$',
+            items['email'])
+        if match is None:
+            self.dataError = {'result': 'Invalid data email'}, 405
+            return(self.dataError)
+        else:
+            return False
+
+
+class Login(CheckUserData):
+    def __init__(self, data):
+        self.data = data
+        self.WhiteSpace = CheckUserData.checkWhiteSpace(self, self.data)
+        self.empty = CheckUserData.checkEmptyData(self, self.data)
```

Comment:
There's inconsistency in naming variables. Some are camelCased, some snake_cased and some have the first letter uppercase. Consider refactoring.